ivadomed/inference.py

```diff
@@ -127,20 +127,30 @@ def segment_volume(folder_model, fname_images, gpu_number=0, options=None):
     # Load model training config
     context = imed_config_manager.ConfigurationManager(fname_model_metadata).get_config()
 
-    if options is not None and any(pp in options for pp in ['thr', 'largest', ' fill_holes', 'remove_small']):
+    postpro_list = ['binarize_prediction', 'keep_largest', ' fill_holes', 'remove_small']
```

Comment:
Names from SCT now fits the ivadomed names (see: https://github.com/neuropoly/spinalcordtoolbox/pull/3090)